### PR TITLE
feat(travis): hook up docs to travis

### LINKED ISSUE #28: hook up docs to travis
Right now the docs are not covered by travisCI, but it certainly would be useful to at least run a [markdown linter](https://github.com/mivok/markdownlint) on the docs.

### PR DESCRIPTION
**Pull request by bacongobbler**
closes #28

### CODE REVIEW

File: docs/Makefile [hunk 1]
```diff
@@ -16,7 +16,7 @@ clean:
 	rm -rf $(BUILDDIR)/*
 
 deps:
-	pip install --disable-pip-version-check -r requirements.txt
```
Comment (mboersma) [inline docs/Makefile]: Why was this removed? Without that flag you get the annoying upgrade message all the time.

Comment (bacongobbler) [inline docs/Makefile]: just sticking to being consistent with lines 31 and 32 in .travis.yml. I can revert this if you'd prefer :)

Comment (mboersma) [inline docs/Makefile]: Oh I see. Now I'm torn between improving consistency and reducing pointless log messages...

Meh, this is trivial either way, LGTM.
